_posts/2022-06-08-ask-the-hybrid-brains.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Ask the Hybrid Brains, Spring 2022"
+summary: "Our first in-person hybrid Ask The Brains event"
+date: 2022-06-08T19:15:00
+meetup: TODO
+speakers:
+- name: Everyone
+image:
+  url: /img/talks/hybrid-brains.jpg
+  title: Ask the Hybrid brains
+tags:
+- workshop
+- questions
+- collaboration
+- virtual
+layout: event.html
+collection: events
+online: true
+venue:
+  name: PLATF9RM
+  link: https://www.platf9rm.com/
+  location: https://goo.gl/maps/pkoNTTRA27bh5GKD8
+  address: Floor 5, Tower Point, 44 North Road, Brighton, BN1 1YR
+  latlong: 50.825944,-0.1437302
+# draft: true # Delete this line to publish
+---
+We're back! You will be happy to know that you will finally have the opportunity to come face-to-face with like-minded developers and enthusiasts (at Async at least) for the first time in over 2 years! Can't make it in-person? Don't worry, you'll also be able to join us online.
```

Comment:
"You will be happy" - sounds forceful! 

I would consider rephrasing it along the lines of "we are happy to announce" although that's also how the next paragraph starts, so...